docs/src/pages/guides/FHIRServerUsersGuide.md

```diff
@@ -1098,6 +1099,121 @@ team can more easilly read the messages.
 It is also possible to configure the persistence properties for a specific tenant, for example to set an alternate
 database hostname or database schema name.
 
+## 4.10 BulkData
+### 4.10.1 BulkData Export
+BulkData Export is implemented according to [FHIR r4 BulkData spec](http://hl7.org/fhir/uv/bulkdata/STU1/export/index.html).   
+There are 2 modules involved inside the implementation:
+- fhir-operation-bulkdata
+- fhir-bulkimportexport-webapp   
+
+To integration test BulkData, there are tests in ExportOperationTest.java in fhir-server-test module with server integration test cases for system, patient and group export.  
+The *fhir-operation-bulkdata* project implements the REST APIs for Bulkdata export as FHIR operations.  There are three operations ExportOperation - system export, PatientExportOperation - Patient export and GroupExportOperation - group export.  Each operation calls the JavaBatch framework defined in the *fhir-bulkimportexport-webapp* project to execute the export unit-of-work.   
+There are 3 chunk style JavaBatch jobs defined as following in *fhir-bulkimportexport-webapp* project for the above 3 export operations:  
+
+- FhirBulkExportChunkJob
+- FhirBulkExportPatientChunkJob
+- FhirBulkExportGroupChunkJob
+
+The *fhir-bulkimportexport-webapp* module is a wrapper for the whole BulkData web application, which is the build artifact - fhir-bulkimportexport.war.   This web archive is copied to the apps directory of the liberty fhir-server instance. Following is a sample liberty server configuration (server.xml) for fhir-bulkimportexport.war:
+
+```xml
+    <webApplication id="fhir-bulkimportexport-webapp" location="fhir-bulkimportexport.war" name="fhir-bulkimportexport-webapp">
+        <classloader commonLibraryRef="fhirSharedLib" privateLibraryRef="configResources,fhirUserLib"/>
+        <application-bnd>
+            <security-role id="users" name="FHIRUsers">
+                <group name="FHIRUsers"/>
+            </security-role>
+        </application-bnd>
+    </webApplication>
+```
+
+BulkData web application writes the exported FHIR resources to IBM COS or Amazon S3 bucket, as configured in the per-tenant bulkdata.json configuration file.  The bulkdata.json file is stored in the tenant configuration directory of each fhir-server instance. The following is a bulkdata.json which is configured to export the FHIR resources into fhir-bulkdata-sample bucket of IBM COS: 
+
+```json
+{
+  "applicationName": "fhir-bulkimportexport-webapp",
```

Comment:
these elements should be described in a table. 

Parameter Name   | Description
|--------------| ------------|
`applicationName`| fixed value |
`moduleName`| fixed value |
`jobParameters.cos.bucket.name`| fixed value |
`jobParameters.cos.location`| fixed value |
`jobParameters.cos.endpointurl`| fixed value |
`jobParameters.credential.ibm`| fixed value |
`jobParameters.cos.api.key`| fixed value |
`jobParameters.cos.srvinst.id`| fixed value |
`implementation_type`| fixed value |
`batch-uri`| fixed value |
`batch-user`| fixed value |
`batch-user-password`| fixed value |
`batch-truststore`| fixed value |
`batch-truststore-password`| fixed value |
`serverHostname`| fixed value |
`contextRoot`| fixed value |

Reply:
you can copy the above table below the JSON, I hope it helps

Reply:
the description for the others parameters are very descriptive, so I didn't add comments for them. but yes, make sense to add them all. I have just added the table to the document